### DISCUSSION TITLE
About the usage of "void main()"

### DISCUSSION THREAD

**Discussion by Philos0303** [kgabis/parson#183] [Show and tell]
Hello, it's my great honor to view your code. But during the running in my IDE, I met a bug and wanna share. =>{
    It's supported C99 in my IDE ,so when I  run  the  command of 'make' ,it shows me "error: return type of ‘main’ is not ‘int’ [-Wmain]". I  search the info, then  found , "void main()" is a wrong usage, maybe  it is  supposed in C89, but  it's overtime.
} =>{
 I  hope you could change your code in  the branch of master,  in  the  file  of  "main.c" ,use "int main(){}"  instead  of  "void main(){}". Then  your  code  will  be  more  friendly  to  a  freshman  to  run.

**Comment by kgabis** [comment]
I'm not using ```void main(...)``` anywhere, nor do I have a main.c file.

**Reply by kgabis** [reply]
parson的代码没有```void main```， 请查一下你的。有可能```void main```在你的代码。
